pontoon/base/views.py

```diff
@@ -168,111 +166,137 @@ def authors_and_time_range(request, locale, slug, part):
     }, safe=False)
 
 
+def _get_entities_list(locale, project, form):
+    """Return a specific list of entities, as defined by the `entity_ids` field of the form.
```

Comment:
Maybe?

Reply:
More seriously, I think convention is to always have a short description on the same line as the quotes, then add more explanations after a new line. Like this: 

```python
def foo(a, b):
    """Return something that is very useful.

    The reason why we do this is because bar. And we do it in this particular 
    way, because.

    :arg a: list of things we care about
    :arg b: string that contains something valuable

    """
    pass
```

Reply:
See for example the way @willkg writes his comments in Antenna: https://github.com/mozilla-services/antenna/blob/master/antenna/util.py

Btw @willkg, do you use a tool to generate API docs automatically from those comments? If so, which one?

Reply:
I would be in favor of https://www.python.org/dev/peps/pep-0257/ or any lint-able format.

Reply:
There's a PEP for formatting docstrings that I tend to just follow because I'm used to doing it that way:

https://www.python.org/dev/peps/pep-0257/

That talks about text in the docstring particularly conventions around the first line. Then I couple that with this:

http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists

which is how I do parameters, parameter types, exceptions raised, returned values, and other things.

Doing it that way works nicely with Sphinx autodoc (``sphinx.ext.autodoc``).

Hope that helps!

Reply:
Thanks @willkg @jotes @adngdb, this looks great!

Let's keep this particular comment unchanged and follow the standard.